Lib/typing.py

```diff
@@ -999,16 +999,42 @@ def __dir__(self):
 
 
 class _GenericAlias(_BaseGenericAlias, _root=True):
-    def __init__(self, origin, params, *, inst=True, name=None,
+    """The type of parameterized generics.
```

Comment:
I wonder if this should be a docstring or a comment -- it's not a public class.

Reply:
Oh, interesting, I wasn't aware of that convention. I've gone ahead and changed these docstrings to comments.

Reply:
I am super sorry for making you do that! It seems the rest of this module uses docstrings for all internal types.

As penance I will fix it back before landing.